AdobeFlashPlayer/AdobeFlashURLProvider.py

```diff
@@ -34,15 +35,25 @@ class AdobeFlashURLProvider(Processor):
     """Provides URL to the latest Adobe Flash Player release."""
     description = __doc__
     input_variables = {
-        "url": {
-            "required": False,
+        'url': {
```

Comment:
I don't feel strongly about single quotes vs double quotes, but consistency is nice. The existing code used double quotes here, so maybe your changes should as well.

Reply:
Okay. I can change that. You're right that consistency is good. I think my excitement at getting this working overrode my better judgement when reviewing my changes. I'll change them to double quotes.